src/main/java/marquez/service/models/Namespace.java

```diff
@@ -43,4 +44,11 @@ public Namespace(
     this.ownerName = ownerName;
     this.description = description;
   }
+
+  public Namespace(
+      String name, String ownerName, String description) {
+    this.name = name;
```

Comment:
You'll also want to set the `guid` and `createdAt` to `null` in the constructor:

```java
this.guid = null;
this.name = name;
this.createdAt = null;
this.ownerName = ownerName;
this.description = description;
```

This isn't ideal, but we are working towards cleaning up our service models